MapboxNavigation/PollyVoiceController.swift

```diff
@@ -96,7 +95,11 @@ public class PollyVoiceController: RouteVoiceController {
             input.voiceId = voiceId
         }
         
-        input.text = "<speak><prosody volume='\(instructionVoiceVolume)' rate='\(instructionVoiceSpeedRate)'>\(text)</prosody></speak>"
+        let wrappedSSML = text
+            .replacingOccurrences(of: "<speak>", with: "<speak><prosody volume='\(instructionVoiceVolume)' rate='\(instructionVoiceSpeedRate)'>")
+            .replacingOccurrences(of: "</speak>", with: "</prosody></speak>")
```

Comment:
With my security hat on, this seems like it would enable someone to break the navigation SDK simply by renaming a road to something like `<speak>` or `</speak>`. Using a proper XML parser would be best, although if we’re absolutely certain that the Directions API will be giving us a document with `<speak>` as the root element, then we could address the security concern by restricting finding-and-replacing only at the beginning and end of the string.